dev/ProgressBar/ProgressBar_themeresources.xaml

```diff
@@ -1,288 +1,46 @@
-            <StaticResource x:Key="ProgressBarBackground" ResourceKey="SystemControlBackgroundBaseLowBrush" />
-            <StaticResource x:Key="ProgressBarBorderBrush" ResourceKey="SystemControlForegroundBaseHighBrush" />
+
+            <StaticResource x:Key="ProgressBarForeground" ResourceKey="SystemColorHighlightColor" />
+            <StaticResource x:Key="ProgressBarBackground" ResourceKey="SystemColorWindowColor" />
+            <StaticResource x:Key="ProgressBarBorderBrush" ResourceKey="SystemColorWindowTextColor" />
+            <StaticResource x:Key="ProgressBarPausedForegroundColor" ResourceKey="SystemColorWindowTextColor" />
+            <StaticResource x:Key="ProgressBarErrorForegroundColor" ResourceKey="SystemColorHotlightColor" />
         </ResourceDictionary>
+
         <ResourceDictionary x:Key="Light">
-            <x:Double x:Key="ProgressBarIndicatorPauseOpacity">0.6</x:Double>
-            <x:Double x:Key="ProgressBarThemeMinHeight">4</x:Double>
             <Thickness x:Key="ProgressBarBorderThemeThickness">0</Thickness>
-            <Color x:Key="SystemControlErrorBackgroundColor">#29C50500</Color>
-            <!-- Hex value for SystemErrorText at 0.16 opacity -->
-            <contract7NotPresent:Color x:Key="SystemErrorTextColor">#C50500</contract7NotPresent:Color>
-            <contract7NotPresent:SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Color="{ThemeResource SystemErrorTextColor}"/>
+
             <StaticResource x:Key="ProgressBarForeground" ResourceKey="AccentAAFillColorDefaultBrush" />
-            <StaticResource x:Key="ProgressBarBackground" ResourceKey="ControlFillColorTertiaryBrush" />
+            <StaticResource x:Key="ProgressBarBackground" ResourceKey="ControlAAStrokeColorDefault" />
             <StaticResource x:Key="ProgressBarBorderBrush" ResourceKey="ControlStrokeColorDefaultBrush" />
-            <SolidColorBrush x:Key="ProgressBarBackgroundThemeBrush" Color="#30000000" />
-            <SolidColorBrush x:Key="ProgressBarBorderThemeBrush" Color="Transparent" />
-            <SolidColorBrush x:Key="ProgressBarForegroundThemeBrush" Color="#FF4617B4" />
-            <SolidColorBrush x:Key="ProgressBarIndeterminateForegroundThemeBrush" Color="#FF4617B4" />
+            <StaticResource x:Key="ProgressBarPausedForegroundColor" ResourceKey="SystemFillColorCaution" />
+            <StaticResource x:Key="ProgressBarErrorForegroundColor" ResourceKey="SystemFillColorCritical" />
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+    <x:Double x:Key="ProgressBarMinHeight">3</x:Double>
+    <x:Double x:Key="ProgressBarTrackHeight">1</x:Double>
+    <CornerRadius x:Key="ProgressBarCornerRadius">1.5</CornerRadius>
+    <CornerRadius x:Key="ProgressBarTrackCornerRadius">0.5</CornerRadius>
+
     <Style TargetType="ProgressBar" BasedOn="{StaticResource DefaultProgressBarStyle}" />
+    <Style x:Key="DefaultProgressBarStyle" TargetType="ProgressBar" />
```

Comment:
I'm not sure we should be removing the DefaultProgressBarStyle entirely. It's the style for the WUX ProgressBar, so if we make this blank, anyone using the old ProgressBar will I suppose end up with nothing. I think we should either keep it (and make sure it doesn't look entirely broken) or, at the least, don't include these two lines so that it defaults back to whatever is it generic.xaml.

Reply:
I talked to Karen and she agreed we don't need it (and really we don't, it's not used in the ProgressBar.xaml at all), but I did find a couple of random spots in the enlistment where it's mentioned. @YuliKl what's the possibility of someone really using it? Do we need to keep it?

Reply:
That's fine, but then I think we should delete these two lines, so that we're not deliberately setting it to empty.

Reply:
I did remove them :)

Reply:
Oh hey! I didn't see the new iteration. :D Cool.